util/osx-packages.txt

```diff
@@ -21,5 +22,22 @@ bullet --with-shared
 protobuf
 clang-format
 graphviz
-px4/px4/gcc-arm-none-eabi
-hg
\ No newline at end of file
+
+# firmware - required
+hg
+python
+gcc-arm-none-eabi-49    # requires `brew tap PX4/homebrew-px4`
```

Comment:
`avr-libc` needs to be changed to the brew cask name, and `gcc-arm-none-eabi` reverted back to what it was originally